### ISSUE TITLE
macro concatenate error

### ISSUE THREAD

**Issue by 04cfb1ed** [vks/average#12]
I'm trying to use the example from [https://docs.rs/average/0.9.3/average/macro.concatenate.html]() with average 0.9.3 in rust 1.33.0

```rust
use average::*;
use average::{Estimate, Max, Min};

concatenate!(MinMax, [Min, min], [Max, max]);
```

and I get this error
[rustc] macros that expand to items must either be surrounded with braces or followed by a semicolon

**Comment by 04cfb1ed** [comment]
My mistake I assume that the macro concatenate should go outside the block because of the code that was generated (struct and impl block)

Solved with moving the macro inside a function

**Comment by vks** [comment]
Your fix works, but your initial example should work as well. Fortunately, this is easy to fix, stay tuned.

**Comment by vks** [comment]
@04cfb1ed I published 0.9.4, fixing your original issue. Thanks for the report!
